sssom/util.py

```diff
@@ -187,7 +192,9 @@ def dataframe_to_ptable(df: pd.DataFrame, priors=[0.02, 0.02, 0.02, 0.02], inver
         elif p == 'dbpedia-owl:different':
             pi = 3
         else:
-            raise Exception(f'Unknown predicate {p}')
```

Comment:
Why?

Reply:
Discussed this with Chris. As of now the lines above are addressing `p = ` hardcoded predicates (handful) . I do not completely understand that part of the code completely. This is a temporary fix I made because I had used new predicates as a test due to which it was `throw`ing errors. For e.g. `!owl:equivalentClass`